app/views/product_drives/show.html.erb

```diff
@@ -100,3 +100,48 @@
     </div>
   </div>
 </section>
+
+<section class="content">
```

Comment:
Can we add a check in the request spec for this to see that the right information is being displayed? If there isn't a request spec, we should create one :)

Reply:
I have added a test in my latest commit.